exercises/concept/treasure-chest/src/TreasureChest.elm

```diff
@@ -0,0 +1,28 @@
+module TreasureChest exposing (..)
+
+{-| One. define a single variant generic TreasureChest custom type with associated data:
+
+  - a String (for a password)
+
+        - a generic (for the treasure)
+
+-}
+
+
+type TreasureChest
+    = TreasureChest
```

Comment:
```suggestion
    = TODO
```

Reply:
Yep, this is a good idea. This still compiles so elm-format is happy, but the message to the student is clear ...